### DISCUSSION TITLE
Portuguese Brazil/Portugal flavours mixed

### DISCUSSION THREAD

**Discussion by marekoid** [argosopentech/argos-translate#361] [General]
Is Portuguese language model supposed to pt_PT or pt_BR? It seems to be mixing them making it effectively not fit to be used both when needing European or Brazilian flavour. The issue is that some words are completely different and a person from one region may not understand those coming from the other region. Would it be possible to have explicitly Brazilian and European flavours like it is the case in MS Translator?

**Comment by ghost** [comment]
Recently, I encountered the same situation when dealing with the `pt_BR` language. The Argon package only supports the `pt` tag, which completely rendered my methodology unviable. This lack of sensitivity to the language leads to many grammatical errors and even drastic changes in some words. It is frustrating to see how the lack of proper support can compromise the quality and accuracy of work in projects that require greater attention to linguistic variations.
